.github/workflows/copyright-update.yml

```diff
@@ -15,4 +15,11 @@ jobs:
       - uses: FantasticFiasco/action-update-license-year@771ff9afdc55b09e1fb649cf03e312d0cf86b4a6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          transform: (?<from>\d{4})+-?(\d{4})?
\ No newline at end of file
+          transform: (?<from>\d{4})+-?(\d{4})?
+          path: |
+            LICENSE
+            examples/async_example.py
+            examples/custom_vocabularies_example.py
+            examples/generator_streaming_example.py
+            examples/language_identification_example.py
+            examples/topic_extraction_example.py
```

Comment:
After investigating found a solution but not sure if this is what we want @amikofalvy but since `microphone_streaming_example.py` is a special case, we can't just use `examples/*.py` and have to update this with every example file we want to update license year for.

Reply:
I think we should continue this investigation in a separate PR since it is not related to Language ID specifically